src/main/java/com/google/devtools/build/lib/exec/StandaloneTestStrategy.java

```diff
@@ -366,6 +366,8 @@ private static BuildEventStreamProtos.TestResult.ExecutionInfo extractExecutionI
     BuildEventStreamProtos.TestResult.ExecutionInfo.Builder executionInfo =
         BuildEventStreamProtos.TestResult.ExecutionInfo.newBuilder();
 
+    executionInfo.setExitCode(spawnResult.exitCode());
```

Comment:
The comment under `SpawnResult#exitCode()` notes that this value is only actually meaningful in certain circumstances. *But*, the new code is called only after a block that throws if `!spawnResult.setupSuccess()`. So this is actually correctly placed.

Can you just add a comment like `// spawnResult.exitCode() is always meaningful because this code only runs if spawnResult.setupSuccess()`?